charts/gardener/provider-local/extension/values.yaml

```diff
@@ -65,6 +65,7 @@ webhookConfig:
   servicePort: 443
   serverPort: 10250
 
+metricsPort: 8080
```

Comment:
Can you use this value in? https://github.com/gardener/gardener/blob/d54863027b10e841a72ed23bc7255a933b6d98ea/charts/gardener/provider-local/extension/templates/deployment.yaml#L31

Reply:
Yep, done.